src/main/resources/model/instance.json

```diff
@@ -552,6 +569,12 @@
       "index": "keyword_lowercase",
       "processor": "sortTitleProcessor"
     },
+    "sort_date1": {
+      "searchTypes": [ "filter", "sort" ],
+      "type": "search",
+      "index": "keyword_lowercase",
```

Comment:
I think we should make it an `number` type to make sure that we will have correct number-based order.

Reply:
updated